backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/skeleton/BpkSkeleton.kt

```diff
@@ -217,7 +217,7 @@ fun BpkHeadlineSkeleton(
     Box(
         modifier = modifier
             .enhanceHeadlineHeight(skeletonHeightSize)
-            .background(shimmerBackgroundColor(), RoundedCornerShape(BpkBorderRadius.Sm)),
+            .background(shimmerBackgroundColor(), RoundedCornerShape(BpkBorderRadius.Xs)),
```

Comment:
almost :) the small headline size has a radius of 2dp, so we need to change it based on the size as well

Reply:
fixed, screenshot updated